src/functions/getResult/framework/handler.ts

```diff
@@ -23,8 +23,8 @@ export async function handler(event: APIGatewayEvent, fnCtx: Context): Promise<R
 
     const validationResult =
       joi.validate({
-        staffNumber: appRefPathParam,
-        appRef: staffNumberParam,
+        staffNumber: staffNumberParam,
```

Comment:
What was this all about?! 😅

Reply:
Yeah I found this yesterday when fixing tests. Had a chat with Ahmed and it seems to have been a mistake that was missed by the tests because the validation is similar and we use similar numbers for testing staffNumber and application reference `e.g. 123456`. So we were validating staffNumber with appRef validation rules and vice versa.